### PR TITLE
Made signup and login page

### PR DESCRIPTION
**Pull request by poojavishnoi**
I have pulled the files from wishlist by mistake.So please check signup and login page files.

### CODE REVIEW

**Comment by anandkumarcoder** [comment]
try to add preview link for easy review

File: css/signup.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+body{
+  background-color: rgb(248, 248, 248);
```
Comment (anandkumarcoder) [inline css/signup.css]: use consistent color assigning values , below you have used variables

**Review by anandkumarcoder — APPROVED** [review]
looks good for meerging

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
```
Comment (zabihshaik) [inline cart.html]: Better to add meta tag with author and description

**Review by zabihshaik — APPROVED** [review]
Everything looks to good. 
UI Looks cool.

File: cart.html (hunk 2 above)
Comment (poojavishnoi) [reply cart.html]: Yup will add that at the end

File: css/signup.css (hunk 1 above)
Comment (poojavishnoi) [reply css/signup.css]: done

**Pull request merged**